server/channels/api4/upload.go

```diff
@@ -17,9 +17,9 @@ import (
 )
 
 func (api *API) InitUpload() {
-	api.BaseRoutes.Uploads.Handle("", api.APISessionRequired(createUpload)).Methods("POST")
+	api.BaseRoutes.Uploads.Handle("", api.APISessionRequired(createUpload, handlerParamFileAPI)).Methods("POST")
```

Comment:
This is not necessary since this endpoint is just about creating an upload session so it can be bound by normal request payload limits.